src/devices/Hx711/README.md

```diff
@@ -0,0 +1,141 @@
+This can be handy for creating your own industrial scale, process control, or simple presence detection.
+
+## Documentation
+
+**[HX711 - Source 1](https://homotix_it.e-mind.it/upld/repository/File/hx711.pdf)**
+
+**[HX711 - Source 2](https://html.alldatasheet.com/html-pdf/1132222/AVIA/HX711/457/1/HX711.html)**
+
+**[Example for Arduino](https://www.moreware.org/wp/blog/2020/06/09/arduino-come-funziona-la-board-per-celle-di-carico-hx711/)**
+
+## Usage with calibration process
+```C#
+int pinDout = 23;
+int pinPD_Sck = 24;
+
+using (var controller = new GpioController())
+{
+    using (var hx711 = new HX711(pinDout, pinPD_Sck, gpioController: controller, shouldDispose: false))
+    {
+        hx711.PowerUp();
+        Console.WriteLine("HX711 is on.");
+
+        for (int i = 0; i < 3; i++)
+        {
+            Console.WriteLine("Known weight (in grams) currently on the scale:");
+            var weightCalibration = int.Parse(Console.ReadLine() ?? "");
+            hx711.SetCalibration(Mass.FromGrams(weightCalibration));
+        }
+
+        Console.WriteLine("Press ENTER to tare.");
+        _ = Console.ReadLine();
+        hx711.Tare();
+        Console.WriteLine($"Tare set. Value: {hx711.TareValue}");
+
+        Console.WriteLine("Press ENTER to start reading.");
+        _ = Console.ReadLine();
+
+        for (int i = 0; i < 25; i++)
+        {
+            var weight = hx711.GetWeight();
+            Console.WriteLine($"Weight: {weight}");
+
+            Thread.Sleep(2_000);
+        }
+
+        hx711.PowerDown();
+        Console.WriteLine("HX711 is off.");
+
+        Console.WriteLine("Press ENTER to close.");
+        _ = Console.ReadLine();
+    }
+}
+```
+
+## Usage without calibration process
+```C#
+int pinDout = 23;
+int pinPD_Sck = 24;
+
+using (var controller = new GpioController())
+{
+    using (var hx711 = new HX711(pinDout, pinPD_Sck, gpioController: controller, shouldDispose: false))
+    {
+        hx711.PowerUp();
+        Console.WriteLine("HX711 is on.");
+
+        // 1 gram every 107.55 hx711 units
+        hx711.SetReferenceUnit(referenceUnit: 107.55);
+
+        Console.WriteLine("Press ENTER to tare.");
+        _ = Console.ReadLine();
+        hx711.Tare();
+        Console.WriteLine($"Tare set. Value: {hx711.TareValue}");
+
+        Console.WriteLine("Press ENTER to start reading.");
+        _ = Console.ReadLine();
+
+        for (int i = 0; i < 25; i++)
+        {
+            var weight = hx711.GetWeight();
+            Console.WriteLine($"Weight: {weight}");
+
+            Thread.Sleep(2_000);
+        }
+
+        hx711.PowerDown();
+        Console.WriteLine("HX711 is off.");
+
+        Console.WriteLine("Press ENTER to close.");
+        _ = Console.ReadLine();
+    }
+}
+```
+
+## Hardware Required
+* Load cell
+* HX711
+* Male/Female Jumper Wires
+
+## Circuit - Scheme
+![Fritz diagram](raspberry_hx711_load_cell.png)
```

Comment:
missing extra enter
Markdown 101: each title should be followed by an empty line

Reply:
Ok, i fixed it.
I didn't know about this rule, sorry.